src/Phoenix.vue

```diff
@@ -2,7 +2,17 @@
   <div class="uk-height-1-1">
     <skip-to target="main">Skip to main</skip-to>
     <div id="Phoenix" class="uk-height-1-1">
-      <template v-if="!showHeader">
+      <div
+        v-if="user.isAuthenticated && !user.userReady"
+        class="loading-overlay"
+        :style="{
+          backgroundImage: 'url(' + configuration.theme.loginPage.backgroundImg + ')'
+        }"
+        uk-height-viewport
+      >
+        <oc-spinner size="xxxlarge" :aria-label="$gettext('Loading')" class="uk-position-center" />
```

Comment:
From my personal feeling the spinner is too big. I would choose `xlarge` size and border sizes in the css class `10px`.